extensions/amp-script/0.1/amp-script.js

```diff
@@ -247,6 +247,13 @@ export class AmpScript extends AMP.BaseElement {
       config
     ).then((workerDom) => {
       this.workerDom_ = workerDom;
+      this.workerDom_.onerror = (errorEvent) => {
```

Comment:
yep. its...a worker from worker-dom. It would potentially make sense to just name it `worker`. This is the exact API I plan on changing in https://github.com/ampproject/worker-dom/pull/850